deploy/auditsweeper/delete.sh

```
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Uninstalls print service
+## Usage: ./delete.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+function deleting_auditsweeper() {
+  NS=auditsweeper
+  while true; do
+      read -p "Are you sure you want to delete print helm chart?(Y/n) " yn
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)